packages/odyssey-react/src/components/Text/Text.module.scss

```diff
@@ -0,0 +1,343 @@
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
+  color: $text-body;
+  font-family: $body-font-family;
+  font-size: ($base-font-size / 16px) * 100%;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: 0;
+  line-height: $base-line-height;
+  font-feature-settings: "lnum", "pnum";
+  text-decoration-skip-ink: all;
+
+  /* stylelint-disable selector-max-type, selector-max-class, selector-class-pattern */
+  abbr,
+  &.abbr {
+    border-block-end: 1px dashed $color-primary-dark;
+    text-decoration: none;
+  }
+
+  em,
+  &.em {
+    font-style: italic;
+
+    > em {
+      text-decoration: underline;
+    }
+  }
+
+  strong,
+  &.strong {
+    font-weight: 600;
+
+    > strong,
+    > .strong {
+      text-decoration: underline;
+    }
+  }
+
+  sub,
+  &.sup {
+    font-size: ms(-2);
+    line-height: 1;
+    vertical-align: super;
+  }
+
+  sub,
+  &.sub {
+    font-size: ms(-2);
+    line-height: 1;
+    vertical-align: sub;
+  }
+
+  blockquote,
+  &.blockquote {
+    max-width: $max-line-length;
+    margin-block-start: 0;
+    margin-block-end: $type-margin;
+    margin-inline: 0;
+    padding-block: 0;
+    padding-inline-end: 0;
+    padding-inline-start: $spacing-s;
+    border-inline-start: 3px solid $border-color-display;
+
+    &:last-child {
+      margin-block-end: 0;
+    }
+  }
+
+  p,
+  &.p {
+    max-width: $max-line-length;
+    margin-block-start: 0;
+    margin-block-end: $type-margin;
+    margin-inline: 0;
+
+    details &,
+    .details & {
+      font-size: ms(0);
+    }
+
+    &:last-child {
+      margin-block-end: 0;
+    }
+  }
+
+  pre,
+  &.pre {
+    margin-block-start: 0;
+    margin-block-end: $type-margin;
+    margin-inline: 0;
+    font-family: $mono-font-family;
+    white-space: pre-wrap;
+    tab-size: 2;
+
+    &:last-child {
+      margin-block-end: 0;
+    }
+  }
+
+  cite,
+  &.cite {
+    font-style: italic;
+  }
+
+  del,
+  &.del {
+    display: inline-block;
+    background: $color-danger-light;
+  }
+
+  dfn,
+  &.dfn {
+    font-style: italic;
+  }
+
+  ins,
+  &.ins {
+    display: inline-block;
+    background: cv("green", "000");
+  }
+
+  ins::before,
+  ins::after,
+  &.ins::before,
+  &.ins::after {
+    @include is-visually-hidden;
+  }
+
+  ins::before,
+  &.ins::before {
+    content: attr(data-a11y-start);
+  }
+
+  ins::after,
+  &.ins::after {
+    content: attr(data-a11y-end);
+  }
+
+  kbd,
+  &.kbd {
+    @include border-radius($base-border-radius);
+
+    display: inline-block;
+    padding-block: 0;
+    padding-inline: $spacing-xs;
+    border: 1px solid cv("gray", "200");
+    background: cv("gray", "000");
+    box-shadow: 0 2px 0 cv("gray", "000"), 0 3px 0 cv("gray", "200");
```

Comment:
using px values here won't allow our styles to scale with users who need a smaller or larger font size. Consider refactoring to use rem to fix this

Reply:
Borders and box-shadows are some of the few properties that shouldn't scale with our type. However, it'd be good to tokenize these soon rather than use the fixed values here (esp color).

@arnoldsandoval-okta do you know if we have a "review code for non-token values" ticket?